### PR TITLE
#changed ShortcutRecorder converted to ARC and ARM

### PR DESCRIPTION
**Pull request by jamesstout**
## Changes:
Converted old Shortcut code from https://code.google.com/archive/p/shortcutrecorder/source/default/source
to ARC and built for ARM

## Closes following issues:
- 

## Tested:
- [x] iMac / MacBook
- macOS version: 10.15.7 (19H15)
- Xcode version: 12.2 (12B45b)

### CODE REVIEW

File: Frameworks/ShortcutRecorder.framework/Versions/A/Headers/SRRecorderCell.h [hunk 1]
```diff
@@ -12,7 +12,7 @@
 //      Jamie Kirkpatrick
 
 #import <Cocoa/Cocoa.h>
-#import <ShortcutRecorder/SRCommon.h>
+#import "SRCommon.h"
```
Comment (Kaspik) [inline Frameworks/ShortcutRecorder.framework/Versions/A/Headers/SRRecorderCell.h:15]: Is this actually correct with modern framework bridging? I mean, if it works, then ship it, just interesting change to me! :)

**Review by Kaspik — APPROVED** [review]
Love it, waaaaay better! 🚀

File: Frameworks/ShortcutRecorder.framework/Versions/A/Headers/SRRecorderCell.h (hunk 1 above)
Comment (jamesstout) [reply Frameworks/ShortcutRecorder.framework/Versions/A/Headers/SRRecorderCell.h:15]: Yeah I thought that, but, that's what Xcode did. It works.

**Comment by jamesstout** [comment]
Don't know what you want to do with the source code? I've published my changes [here](https://github.com/jamesstout/shortcutrecorder).

**Comment by Kaspik** [comment]
We could probably move the source code under Sequel-Ace org, or just get rid of that? Up to you, if you think it will be helpful in future, let's invite me (email on my Slack profile) to the repo as admin and I'll transfer it! :)

**Review by Jason-Morcos — COMMENTED** [review]
Awesome job, @jamesstout. Love this

**Comment by jamesstout** [comment]
>if you think it will be helpful in future, let's invite me (email on my Slack profile) to the repo as admin and I'll transfer it! :)

Invited

**Pull request merged**